.github/workflows/publish-maven-plugin.yml

```diff
@@ -38,22 +47,33 @@ jobs:
           git config --global user.name 'github-actions[bot]'
           git config --global user.email '41898282+github-actions[bot]@users.noreply.github.com'
 
+
+      # Manually import gpg key so we don't have to deal with environment variable ambiguity.
+      - name: Import GPG Key 
+        run: |
+          rm -rf $HOME/.gnupg/
```

Comment:
We should also probably clean up at the end of  the pipeline with an `always()` step. 🤷 prevents the key from being re-used in other pipelines when we don't want it to.